src/concourse/pipelines/open_edx/mfe/values.py

```diff
@@ -115,6 +116,7 @@
         studio_domain="studio.mitxonline.mit.edu",
         support_url="mitxonline.zendesk.com/hc/",
         terms_of_service_url="https://mitxonline.mit.edu/terms-of-service/",
+        accessibility_url="https://accessibility.mit.edu/",
```

Comment:
You will also need to add this as an attribute of the Pydantic model [here](https://github.com/mitodl/ol-infrastructure/blob/main/src/concourse/pipelines/open_edx/mfe/pipeline.py#L37-L57)

Reply:
Yeah, I did that in my second commit https://github.com/mitodl/ol-infrastructure/pull/1370/commits/3d85506700b5aade6da4cb24a61fdb9ccade576d.